frontend/src/pages/IstioConfigList/FiltersAndSorts.ts

```diff
@@ -93,6 +93,14 @@ export const istioTypeFilter: FilterType = {
       id: 'Gateway',
       title: 'Gateway'
     },
+    {
+      id: 'K8sGateway',
+      title: 'K8sGateway'
+    },
+    {
+      id: 'K8sHTTPRoute',
+      title: 'K8sHTTPRoute'
+    },
```

Comment:
Under the hood, Istio creates a `VirtualService` and `istio.Gateway` when a `K8sGateway` and `K8sHTTPRoute` is created. I wonder if we want to filter out those underlying objects?

![Screenshot from 2022-09-19 12-00-43](https://user-images.githubusercontent.com/6226732/191062200-ab4fe743-a24e-44d3-9cfb-72e77bfe4b83.png)

![Screenshot from 2022-09-19 12-00-15](https://user-images.githubusercontent.com/6226732/191062242-176a63ee-8cab-4b48-b03a-1b45aa2e56ec.png)

If so, we could filter out anything with an `internal.istio.io/parent` annotation.

Reply:
Showing those autogenerated objects was added recently based on community issue https://github.com/kiali/kiali/issues/5213, as there was a confusion with registry only objects. So they are listed and shown as read-only..